CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 - Admin Credential
   - Credential Request Management Board Improvements
+- Company Wallet
+  - Use appropriate path to display logo on cards
```

Comment:
Could you please add this as bugfix to the changelog?

Reply:
updated